includes/collection/class-outbox.php

```diff
@@ -296,28 +296,35 @@ public static function maybe_get_activity( $outbox_item ) {
 	/**
 	 * Get the object ID of an activity.
 	 *
-	 * @param Activity $activity The activity object.
+	 * @param Generic_Object $data The activity object.
```

Comment:
Should this be `Base_Object`? My understanding was that `Generic_Object` was just for cases where we needed dynamic properties.

Reply:
It can be an Activity or Object, so Base_Object would drop params. Aside from that, a base Object does not have an object attribute as far as I know!?!